src/container.jl

```diff
@@ -1,31 +1,43 @@
-struct Trace{F}
+struct Trace{F,U,N,V<:Random123.AbstractR123{U}}
```

Comment:
Do we dispatch on `U`, `N` or `V` somewhere? Otherwise (and probably even in this case) this could be simplified to
```julia
struct Trace{F,R<:TracedRNG}
    f::F
    ctask::Libtask.CTask
    rng::R
end
```